synapse/rest/client/versions.py

```diff
@@ -59,9 +59,7 @@ def on_GET(self, request):
                     # also requires `id_server`. If the homeserver is handling 3PID
                     # verification itself, there is no need to ask the user for `id_server` to
                     # be supplied.
-                    "m.require_identity_server": (
-                        self.config.account_threepid_delegate is None
-                    ),
+                    "m.require_identity_server": False,
```

Comment:
this should be in #5968

Reply:
Ah yeah, what's that doing there.

For future reference, this is handled in https://github.com/matrix-org/synapse/pull/5972 now